cirq-google/cirq_google/engine/virtual_engine_factory.py

```diff
@@ -367,3 +370,36 @@ def create_noiseless_virtual_engine_from_latest_templates() -> SimulatedLocalEng
     processor_ids = list(MOST_RECENT_TEMPLATES.keys())
     template_names = [MOST_RECENT_TEMPLATES[k] for k in processor_ids]
     return create_noiseless_virtual_engine_from_templates(processor_ids, template_names)
+
+
+def create_noisy_virtual_engine_from_processor_id_and_simulator_class(
```

Comment:
The _from_processor_id_and_simulator_class should be obvious from the arguments, right?
Is there a good name we can use for this?  Like, create_default_noisy_quantum_virtual_machine?